core/state_transition.go

```diff
@@ -370,33 +366,24 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, failed bo
 		log.Error("Failed to refund gas", "err", err)
 		return nil, 0, false, err
 	}
-	gasUsed := st.gasUsed()
-
-	// Pay tx fee to tx fee recipient and Infrastructure fund
-	totalTxFee := new(big.Int).Mul(new(big.Int).SetUint64(gasUsed), st.gasPrice)
-
-	var recipientTxFee *big.Int
-	if st.infrastructureAccountAddress != nil {
-		infraTxFee := new(big.Int).Div(new(big.Int).Mul(new(big.Int).SetUint64(gasUsed), new(big.Int).Mul(st.gasPriceMinimum, st.infraFraction.Numerator)), st.infraFraction.Denominator)
-		recipientTxFee = new(big.Int).Sub(totalTxFee, infraTxFee)
-		err = st.creditGas(*st.infrastructureAccountAddress, infraTxFee, msg.GasCurrency())
-	} else {
-		log.Error("no infrastructure account address found - sending entire txFee to fee recipient")
-		recipientTxFee = totalTxFee
-	}
-
-	if err != nil {
-		return nil, 0, false, err
-	}
 
+	// Pay tx fee to tx fee recipient and the proposer (coinbase)
 	txFeeRecipient := msg.GasFeeRecipient()
 	if txFeeRecipient == nil {
 		sender := msg.From()
 		txFeeRecipient = &sender
 	}
-	err = st.creditGas(*txFeeRecipient, recipientTxFee, msg.GasCurrency())
 
-	if err != nil {
+	gasUsed := new(big.Int).SetUint64(st.gasUsed())
+	totalTxFee := new(big.Int).Mul(gasUsed, st.gasPrice)
+	// gasUsed * gasPriceMinimun * proposerFractionNumerator / proposerFractionDenominator
```

Comment:
Remove?

Reply:
The intent was to clarify what's the math on the line below.
IMO, constantly using `new(big.Int).` makes the lines harder to read.